unit_tests/TestUtils.h

```diff
@@ -160,7 +160,7 @@ struct TestParams {
     }
 };
 
-bool TestParams::skipSerialTests = true;
+bool TestParams::skipSerialTests = false;
```

Comment:
Same here: Did you change this for testing and then forgot to change it back or do we want this in the dev branch?